src/pages/ProfilePage/ProfilePage.js

```diff
@@ -1,10 +1,13 @@
 import React from "react";
 
+import { BetterCalendar } from "../../components/Calendar/Calendar";
```

Comment:
Maybe we can rename this component and its folder and file to MeetingsCalendar? Do you think this calendar will have other information other than student-advisor meeting dates?

Reply:
I just renamed the file, folder, component to MeetingsCalendar